srtcore/queue.h

```diff
@@ -121,16 +116,28 @@ class CUnitQueue
         int    m_iSize;   // size of each queue
 
         CQEntry* m_pNext;
-    } * m_pQEntry,     // pointer to the first unit queue
-        *m_pCurrQueue, // pointer to the current available queue
-        *m_pLastQueue; // pointer to the last unit queue
+    };
 
-    CUnit* m_pAvailUnit; // recent available unit
+    /// Increase the unit queue size (by @a m_iBlockSize units).
+    /// Uses m_mtx to protect access and changes of the queue state.
```

Comment:
`m_mtx` was deleted?

Reply:
Yes. There is no sense in it with the atomic `m_iFlag`. But I forgot to update the comment :)
Thanks for noticing!